Neos.Neos/Resources/Private/Fusion/Prototypes/MenuData.fusion

```diff
@@ -0,0 +1,41 @@
+prototype(Neos.Neos:MenuData) < prototype(Neos.Fusion:Component) {
+
+  node = null
+  itemCollection = null
+  startingPoint = ${node}
+  entryLevel = ${this.startingPoint ? 0 : 1}
+  maximumLevels = 2
+  renderHiddenInIndex = false
+  // currentLevel = ${q(this.node).parentsUntil(this.startingPoint).count()}
+
+  filter = Neos.Fusion:Loop {
+    items = ${this.filter}
```

Comment:
Where is `this.filter`defined?

Reply:
Should be `filter = ''` by default. My problem with this case was that we have in Neos.Neos:Menu a filter string like `!Vendor.Site:Foo,Neos.Neos:Document`, but for FlowQuery I need `![instanceof Vendor.Site:Foo],[instanceof Neos.Neos:Document]`. Or is there a way with which I can use the initial filter expression?